front/src/components/ProfileView/ProfileView.tsx

```diff
@@ -11,7 +12,21 @@ interface IProfileViewProps {
 }
 
 const ProfileView: React.FC<IProfileViewProps> = ({ profile, children }) => {
-  return <ProfileDrawer profile={profile}>{children}</ProfileDrawer>;
+  const hasMusicEmbed =
+    profile &&
+    'musicLink' in profile &&
+    typeof profile.musicLink === 'string' &&
+    profile.musicLink.length > 0;
+
+  return (
+    <>
+      <Gallery
+        mediaList={profile && profile.gallery}
```

Comment:
Could this be replaced by porfile?.gallery ? It would make the condition a little simplier, idk if you agree ?